helm/nginx-ingress-controller-app/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: v0.25.1
 description: A Helm chart for the nginx ingress-controller
 name: nginx-ingress-controller-app
-version: 0.11.1-[[ .SHA ]]
+version: 0.11.2-[[ .SHA ]]
```

Comment:
```suggestion
version: [[ .Version ]]
```

The templating changes and architect-orb has the magic to decide the version. To push to the production catalog we create a github release which pushes the version tag.

cc @corest